system/jlib/jthread.hpp

```diff
@@ -33,6 +33,23 @@
 // no thread priority handling?
 #endif
 
+// Functions used to reset thread-local context variables, when a threadpool starts
+interface IContextLogger;
+typedef unsigned __int64 LogMsgJobId;
```

Comment:
Is it fair to refactor to LogMsgJobInfoId ?
It helps disambiguate between the jobInfo's identifier, and the actual jobId